index.js

```diff
@@ -382,16 +382,18 @@ if (!nativeBinding || process.env.NAPI_RS_FORCE_WASI) {
 
 if (!nativeBinding) {
   if (loadErrors.length > 0) {
-    throw new Error(
-      `Cannot find native binding. ` +
-        `npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). ` +
-        'Please try `npm i` again after removing both package-lock.json and node_modules directory.',
-      { cause: loadErrors }
-    )
+    const musl = process.platform === 'linux' ? isMusl() : false;
+    // Log actual errors to debug console
+    throw loadErrors[0];
+    // throw new Error(
+    //   `Cannot find native binding for platform=${process.platform}, arch=${process.arch}, musl=${musl} \n` +
+    //     `npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). ` +
+    //     'Please try `npm i` again after removing both package-lock.json and node_modules directory.',
+    //   { cause: loadErrors }
+    // )
   }
   throw new Error(`Failed to load native binding`)
 }
 
 module.exports = nativeBinding
-module.exports.hash = nativeBinding.hash
-module.exports.hashInto = nativeBinding.hashInto
+module.exports.plus100 = nativeBinding.plus100
```

Comment:
what is this and why are the hash function exports gone?

maybe needs a rebuild to regenerate this file